### PR TITLE
refactor: rename to goto

### PR DESCRIPTION
**Pull request by haveiss**

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -11,9 +11,9 @@ builds:
     binary: compass
     flags: [-a]
     ldflags:
-      - -X github.com/odpf/compass/cli.Version={{.Tag}}
```
Comment (mabdh) [inline .goreleaser.yml:14]: This should still be `cli` right?

File: .goreleaser.yml [hunk 2]
```diff
@@ -11,9 +11,9 @@ builds:
...
-      - -X github.com/odpf/compass/cli.BuildCommit={{.FullCommit}}
-      - -X github.com/odpf/compass/cli.BuildDate={{.Date}}
+      - -X github.com/goto/compass/cmd.Version={{.Tag}}
```
Comment (mabdh) [inline .goreleaser.yml:14]: This should still be cli right?

File: .goreleaser.yml [hunk 3]
```diff
@@ -81,5 +81,5 @@ brews:
     install: |-
       bin.install "compass"
     commit_author:
-      name: Ravi Suhag
-      email: suhag.ravi@gmail.com
+      name: Suhas Karanth
+      email: suhas.karanth@gojek.com
```
Comment (mabdh) [inline .goreleaser.yml:85]: we could use bot credentials for more neutral commit

```
    commit_author:
      name: github-actions[bot]
      email: 41898282+github-actions[bot]@users.noreply.github.com
```

File: NOTICE [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Change(s)
```
Comment (mabdh) [inline NOTICE:1]: Not sure if this is needed

File: cli/help.go [hunk 5]
```diff
@@ -7,8 +7,8 @@ import (
 var envHelp = map[string]string{
 	"short": "List of supported environment variables",
 	"long": heredoc.Doc(`
-			ODPF_CONFIG_DIR: the directory where compass will store configuration files. Default:
-			"$XDG_CONFIG_HOME/odpf" or "$HOME/.config/odpf".
+			GOTO_CONFIG_DIR: the directory where compass will store configuration files. Default:
```
Comment (mabdh) [inline cli/help.go:10]:
```suggestion
			GOTOCOMPANY_CONFIG_DIR: the directory where compass will store configuration files. Default:
```

File: cli/help.go [hunk 6]
```diff
@@ -7,8 +7,8 @@ import (
...
+			"$XDG_CONFIG_HOME/goto" or "$HOME/.config/goto".
```
Comment (mabdh) [inline cli/help.go:11]:
```suggestion
			"$XDG_CONFIG_HOME/gotocompany" or "$HOME/.config/gotocompany".
```

File: NOTICE (hunk 4 above)
Comment (mabdh) [reply NOTICE:1]: I think lets just add it

Comment (ravisuhag) [reply NOTICE:1]: @mabdh This notice can not be added globally. Only new files can be added a new license or copyright.

Comment (mabdh) [reply NOTICE:1]: @haveiss lets remove this file mas?

File: .goreleaser.yml (hunk 2 above)
Comment (haveiss) [reply .goreleaser.yml:14]: nice catch, will resolve this, there's an issue with the cherry-picking

**Pull request merged**